src/collections/_documentation/accounts/pricing.md

```diff
@@ -30,18 +30,16 @@ On-demand capacity is billed at the end of each billing cycle. You are charged p
 
 Reserved capacity allows you to prepay for significant discounts. Your prepaid, reserved capacity is always applied before on-demand capacity. Unused reserved capacity expires at the end of each billing month.
 
-Every Sentry account includes a varying amount of reserved capacity. For example, our developer plan includes 5,000 events per month.
+Every Sentry account includes a varying amount of reserved capacity. For example, our base team plan includes 50k errors, 100k transactions, and 1 GB of attachments per month.
```

Comment:
suggestion: base team -> our team plan starts at

Reply:
Thanks. Will change. And will also cap "Team" as that's our formal name for that plan.